app/controllers/v0/user_action_events_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module V0
+  class UserActionEventsController < ApplicationController
+    service_tag 'identity'
+
+    def index
+      user_verification = current_user.user_verification
+
+      start_date = params[:start_date].present? ? Date.parse(params[:start_date]) : 1.month.ago.to_date
```

Comment:
Should prob do `Date.parse(params[:start_date]).start_of_day`

Reply:
This is breaking some tests locally, I'm trying to figure out why right now. I'm getting empty arrays, likely because the test wasn't structured for this call